dev/tasks/tasks.yml

```
@@ -1110,9 +1110,8 @@ tasks:
     template: docker-tests/github.linux.yml
     params:
       env:
-        ARCHERY_USE_DOCKER_CLI: 0
         UBUNTU: 20.04
-      flags: -e ARROW_SKYHOOK=ON
+      flags: -e ARROW_SKYHOOK=ON -e ARROW_AZURE=OFF -e ARROW_GCS=OFF -e ARROW_S3=OFF -e ARROW_GANDIVA=OFF
```

How about using `>-` and sort them?

```suggestion
      flags: >-
        -e ARROW_AZURE=OFF
        -e ARROW_GANDIVA=OFF
        -e ARROW_GCS=OFF
        -e ARROW_S3=OFF
        -e ARROW_SKYHOOK=ON
```

---

Nice suggestion!